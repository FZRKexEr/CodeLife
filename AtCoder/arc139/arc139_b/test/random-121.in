100
44 10 10 11 2 17
68 17 14 17 7 5
57 10 12 9 16 19
81 12 17 8 14 8
83 5 7 5 16 3
75 1 8 15 2 1
6 17 4 16 8 7
82 12 10 13 2 16
77 10 4 17 19 8
81 7 17 1 17 12
6 19 5 20 20 2
33 18 14 17 16 3
48 5 20 20 20 11
6 2 13 13 10 16
89 6 20 18 14 13
68 13 11 1 14 2
38 5 10 1 5 11
20 14 14 7 14 12
49 9 13 10 3 13
82 13 1 8 7 20
93 1 1 19 12 5
90 9 1 14 19 16
36 2 9 10 13 8
27 13 20 13 10 2
61 13 18 9 20 4
37 9 13 16 7 7
84 12 10 5 2 8
96 12 10 6 11 11
54 20 5 2 19 19
95 3 17 6 11 20
28 16 6 18 11 10
44 6 1 14 3 5
62 6 3 12 10 12
100 19 2 14 16 8
26 13 17 11 16 19
84 14 12 18 16 17
80 11 2 9 1 12
52 17 2 7 16 9
21 9 17 4 11 18
22 6 6 6 6 12
11 8 12 15 7 3
59 3 7 7 1 7
56 18 4 16 15 7
91 20 20 18 11 3
41 10 2 7 4 17
88 11 10 9 11 16
65 17 4 8 19 12
38 3 5 11 12 1
18 2 5 12 19 4
32 17 12 14 19 9
50 13 9 14 8 18
70 9 16 10 19 20
76 9 11 20 1 8
53 2 9 11 14 14
28 15 2 13 7 17
38 3 2 7 14 20
57 20 2 2 14 8
79 2 14 12 18 19
47 15 3 10 12 6
31 18 13 7 13 1
66 19 15 14 1 18
67 19 7 19 12 1
16 1 2 10 6 3
40 7 9 10 16 9
79 12 20 11 18 19
60 2 10 6 3 1
70 6 8 2 1 13
63 8 19 1 10 10
78 9 3 4 5 17
39 20 6 11 1 2
97 3 1 6 13 20
52 15 6 12 19 7
35 12 20 8 3 19
33 14 15 4 10 15
71 15 3 18 17 13
2 17 10 6 4 17
85 15 20 11 7 12
80 11 4 16 6 20
96 7 11 7 5 8
79 10 13 1 1 3
29 8 3 4 15 4
28 3 7 13 17 20
34 9 6 11 4 1
32 18 3 20 10 17
77 7 11 13 19 8
95 2 19 19 18 14
73 17 6 11 17 16
84 9 17 16 19 20
56 15 16 10 8 10
23 17 1 11 10 10
83 14 12 7 16 18
41 3 9 8 3 4
25 1 13 3 12 7
30 9 17 20 8 9
64 13 2 18 15 17
62 18 19 17 20 4
33 11 6 7 4 18
62 17 5 8 10 20
89 12 9 5 9 10
66 3 10 3 6 2
