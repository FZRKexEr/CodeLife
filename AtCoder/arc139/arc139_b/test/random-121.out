52
28
98
54
54
15
39
28
163
61
22
36
102
30
120
14
38
54
25
74
465
190
32
33
75
44
14
88
66
139
31
31
124
110
32
96
28
55
34
42
52
29
65
210
23
88
109
19
43
70
70
134
22
126
122
182
62
179
56
14
83
47
24
76
125
6
19
36
64
18
422
73
46
40
143
12
45
90
69
13
43
80
19
118
56
70
123
211
110
70
105
35
43
84
162
97
12
78
85
24
